sdk/CoreWCF/Azure.CoreWCF.AzureQueueStorage/NuGet.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
```

Comment:
This cannot be included when the PR is merged.   The Azure SDK repository only permits package references from NuGet or the Azure SDK development feed.

Reply:
This was actually the remnant from our previous code. We no longer need the config file. so removed it